packages/slim-dev/zarf.yaml

```diff
@@ -46,6 +46,27 @@ components:
     import:
       path: ../../dist
       name: module
+    charts:
+      - name: module
+        valuesFiles:
+          - ../../src/pepr/values.yaml
+    actions:
+      onDeploy:
+        before:
+          - cmd: ./zarf tools kubectl annotate secret -n pepr-system pepr-uds-core-api-token meta.helm.sh/release-name=module --overwrite || true
```

Comment:
As discussed synchronously, I think we can pull this into a single `src/pepr/zarf.yaml`, potentially along with the `uds-operator-config` chart. Then we can simplify this import a bit and de-dupe the action portion.